easybuild/easyconfigs/s/scikit-image/scikit-image-0.12.3-foss-2016b-Python-3.5.2.eb

```diff
@@ -0,0 +1,45 @@
+easyblock = 'Bundle'
+
+name = 'scikit-image'
+version = '0.12.3'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://scikit-learn.org/stable/index.html'
+description = """Scikit-learn integrates machine learning algorithms in the tightly-knit scientific Python world,
+building upon numpy, scipy, and matplotlib. As a machine-learning module,
+it provides versatile tools for data mining and analysis in any field of science and engineering.
+It strives to be simple and efficient, accessible to everybody, and reusable in various contexts."""
+
+toolchain = {'name': 'foss', 'version': '2016b'}
+
+# this is a bundle of Python packages
+exts_defaultclass = 'PythonPackage'
+
+dependencies = [
+    ('Python', '3.5.2'),
+    ('Qhull', '2015.2'),
+    ('matplotlib', '1.5.3', versionsuffix),
+    ('Pillow', '3.4.2', versionsuffix),
+]
+
+exts_list = [
+    ('networkx', '1.11', {
+        'source_urls': ['https://pypi.python.org/packages/source/n/networkx'],
+    }),
+    (name, version, {
+        'source_urls': ['https://pypi.python.org/packages/source/s/scikit-image'],
+        'modulename': 'skimage',
```

Comment:
please also add checksums here, since we'll require them in contributions relatively soon (cfr. #5001):

```python
exts_list = [
    ('networkx', '1.11', {
        'source_urls': ['https://pypi.python.org/packages/source/n/networkx'],
        'checksums': ['...'],
    }),
    (name, version, {
        'source_urls': ['https://pypi.python.org/packages/source/s/scikit-image'],
        'modulename': 'skimage',
        'checksums': ['...'],
    }),
]
```